sorting/insert_sort.py

```diff
@@ -0,0 +1,24 @@
+"""Insert Sort Implementation."""
+
+
+def insert_sort(lst=[]):
+    """Insert sort."""
+    for i in range(len(lst)):
+
+        k = lst[i]
+        j = i
+
+        while j > 0 and k < lst[j - 1]:
+            lst[j] = lst[j - 1]
+            j -= 1
+
+        lst[j] = k
```

Comment:
This line isnt necessary since you're already reassigning `lst[j]`inside the while loop